changelog/changelog.py

```diff
@@ -258,7 +258,7 @@ def main():
             changelog.write('\n'.join(lines))
 
             # write back rest of changelog
-            changelog.writelines(original)
+            changelog.writelines(original[1:])
```

Comment:
Looks like there's one fewer newline than there should be to separate the sections. Results of `--update --name CHANGES.md neuropoly/spinalcordtoolbox`:

```
# Changelog

## next-release (2020-11-16)
[View detailed changelog](https://github.com/neuropoly/spinalcordtoolbox/compare/5.0.1...next-release)

**DOCUMENTATION-INTERNAL**

 - Add script to automate requirements-freeze.txt generation.  [View pull request](https://github.com/neuropoly/spinalcordtoolbox/pull/3042)
## 5.0.1 (2020-11-12)
[View detailed changelog](https://github.com/neuropoly/spinalcordtoolbox/compare/5.0.0...5.0.1)
```

Reply:
fixed